src/teacher/dto/teacher.dto.ts

```diff
@@ -95,7 +95,7 @@ export class TeacherDto {
     description: "The subjectId of the teacher",
   })
   @Expose()
-  subjectIds: [String];
+  subjectIds: string;
```

Comment:
Why did we change this ?

Reply:
Now I've solved this pushing new changes

Reply:
@coolbung sir please review and merge